### PR TITLE
Reintroduce raw location updates to the publisher

### PR DESCRIPTION
**Pull request by KacperKluka**
We are reintroducing the option to publish raw location updates. However, this time as a debug tool, thus a flag to the presence data of publishers was also added. I've updated and created the appropriate schemas to keep them up to date.

### CODE REVIEW

**Comment by QuintinWillison** [comment]
I'm requesting additional reviewers. @owenpearson, because he's had more experience with JSON schema than I have. @ikbalkaya, for a fresh pair of eyes on these schemas.

**Review by owenpearson — APPROVED** [review]
LGTM 👍

File: test-resources/geo/presence-data-schema.json [hunk 1]
```diff
@@ -32,6 +32,11 @@
       },
       "additionalProperties": false,
       "required": ["accuracy", "desiredInterval", "minimumDisplacement"]
+    },
+
+    "rawLocations": {
+      "type": ["boolean", "null"],
```
Comment (QuintinWillison) [inline test-resources/geo/presence-data-schema.json:38]: I would be interested to understand why we need to support `null` as a valid value here. If I read correctly, this is effectively tri-state - in which case I don't think the `description` adequately captures what that third state conveys.

Comment (KacperKluka) [reply test-resources/geo/presence-data-schema.json:38]: As the `PresenceData` schema is used in both Subscriber and Publisher SDKs we allow sending `null` when the `rawLocations` does not belong to the specific `type` (being subscriber or publisher). Also it makes it easier in terms of backwards compatibility I think :thinking:

Comment (QuintinWillison) [reply test-resources/geo/presence-data-schema.json:38]: ok, makes sense. Could you consider adding that explanation to the `description` for this `rawLocations` node?

Comment (KacperKluka) [reply test-resources/geo/presence-data-schema.json:38]: https://github.com/ably/ably-asset-tracking-common/pull/30/commits/aa09c07fcf32a6c42997d5a477509a402c474736

**Pull request merged**